std/experimental/allocator/mmap_allocator.d

```diff
@@ -5,8 +5,8 @@ module std.experimental.allocator.mmap_allocator;
 /**
 
 Allocator (currently defined only for Posix and Windows) using
-$(D $(LINK2 https://en.wikipedia.org/wiki/Mmap, mmap))
-and $(D $(LUCKY munmap)) directly (or their Windows equivalents). There is no
+$(LINK2 https://en.wikipedia.org/wiki/Mmap, `mmap`)
+and $(LUCKY munmap) directly (or their Windows equivalents). There is no
```

Comment:
I wasn't sure how to deal with the LUCKY reference here, maybe use the link to Wikpedia as well?

![image](https://cloud.githubusercontent.com/assets/4370550/23589618/9674ecb6-01d0-11e7-8bd6-8df06f6dbe5e.png)